script/Ciel/Day 14/14_01B_01_CIEL14B_KRAKE.txt

```diff
@@ -472,10 +509,11 @@ The Executor soars through the sky.
 [sha:be903b38041d2a2756686dc9ea725dd723c24718]{
 -- Page 39, Offset 38440.
 -- 総耶の街を包む、絶望的な世界の檻。
-A cage confining a hopeless world surrounds the city of Souya.
+// Is 世界の檻 supposed to be a term?
+A cage of worldly despair has descended over the city of Souya.
```

Comment:
not really, but it refers to the Event Storage, which is the planet's ability to shut out the human texture
>The despair of the world's cage has engulfed Souya.